src/ansys/fluent/core/meshing/meshing_workflow.py

```diff
@@ -297,3 +297,33 @@ def __init__(
         self._meshing = meshing
         self._unsubscribe_root_affected_callback()
         self._create_workflow()
+
+
+class CurrentWorkflow(Workflow):
```

Comment:
The problem is that _current_ is not a workflow type. An instantiated workflow object of type `CurrentWorkflow` has none of the specific characteristeristics of the true workflow type.